livox_ros_driver/livox_ros_driver/livox_ros_driver_nodelet.cpp

```diff
@@ -194,9 +195,7 @@ namespace livox_ros
       poll_freq = 2000;
     }
 
-    const ros::Timer poll_lidars = private_node_handle.createTimer(ros::Duration(ros::Rate(poll_freq)), &LivoxRosDriverNodelet::proccessLidarLoop, this);
-
-    return;
+    poll_lidars_ = private_node_handle.createTimer(ros::Duration(ros::Rate(poll_freq)), &LivoxRosDriverNodelet::proccessLidarLoop, this);

```

Comment:
do we need to save this timer as a class member?

Reply:
Yes, otherwise the timer callback is never called.